### PR TITLE
update changelog and add checkout info in migration

### PR DESCRIPTION
**Pull request by AmitPhulera**
https://dimagi-dev.atlassian.net/browse/SAAS-13717

Add information about checking out the commit which would have the migration code. A part of https://github.com/dimagi/commcare-hq/pull/32086

### CODE REVIEW

**Review by dannyroberts — APPROVED** [review]

File: changelog/0051-historical-auditcare-data-migration.yml [hunk 1]
```diff
@@ -16,11 +16,14 @@ update_steps: |
   $ cchq <env> ssh django-manage:0
   $ tmux new -s auditcare
   $ sudo -iu cchq
-  $ cd /home/cchq/www/<env>/current && source python_env/bin/activate
+  $ cd /home/cchq/www/<env>/current
+  $ git checkout f0445cf724e3e2ba3b1b149705a62f4e66ad75b4
```
Comment (millerdev) [inline changelog/0051-historical-auditcare-data-migration.yml:20]: Should this version be assigned to the `min_commcare_version` key?

Comment (AmitPhulera) [reply changelog/0051-historical-auditcare-data-migration.yml:20]: I thought `min_commcare_version` is for Commcare Android Application that's why I have left it blank.

Comment (millerdev) [reply changelog/0051-historical-auditcare-data-migration.yml:20]: I think it's an HQ commit hash, and has nothing to do with Commcare Android. There are quite a few places in commcare-cloud where it says "commcare" and means "commcare hq". Example: `cchq <env> deploy commcare`

Comment (millerdev) [reply changelog/0051-historical-auditcare-data-migration.yml:20]: The comments in the change log template mention HQ explicitly:

https://github.com/dimagi/commcare-cloud/blob/dcec73fbf2f8a323a061204a6fa93407d287ebf2/src/commcare_cloud/manage_commcare_cloud/changelog-template.yml.j2#L4-L7

Comment (AmitPhulera) [reply changelog/0051-historical-auditcare-data-migration.yml:21]: https://github.com/dimagi/commcare-cloud/pull/5439/commits/fa214c19a58ce63a36863fc411dd570ae978fe22

**Review by millerdev — APPROVED** [review]

**Pull request merged**